sql/updates/pre-0.4.2--0.5.0-dev.sql

```diff
@@ -0,0 +1,25 @@
+DROP FUNCTION _timescaledb_internal.ddl_is_change_owner(pg_ddl_command);
+DROP FUNCTION _timescaledb_internal.ddl_change_owner_to(pg_ddl_command);
+
+DROP FUNCTION _timescaledb_internal.chunk_add_constraints(integer);
+DROP FUNCTION _timescaledb_internal.ddl_process_alter_table() CASCADE;
+
+CREATE INDEX ON _timescaledb_catalog.chunk_constraint(chunk_id, dimension_slice_id);
+
+ALTER TABLE _timescaledb_catalog.chunk_constraint
+DROP CONSTRAINT chunk_constraint_pkey,
+ADD COLUMN constraint_name NAME;
+
+UPDATE _timescaledb_catalog.chunk_constraint
```

Comment:
As I mentioned in the previous PR, I don't think there is a safer way to do this using pg_constraint. Your previous PR assumed that there was only one check constraint on a dimension column name, which I believe is /less/ safe.

Reply:
The code that we wrote prior to this update script only tracks a single constraint per dimension in the catalog. I am not sure how the code could have added more constraints without changing the code? Even if it would be possible to have multiple constraints per dimension, I am sure you could still make a query to read that data from `pg_constraint`.

This is about using a source of truth rather than implicit assumptions. I.e., for every `chunk_constraint` row, there must be a corresponding entry in `pg_constraint` with the name of the constraint in it.